lib/rubocop/cop/rails/time_zone.rb

```diff
@@ -105,6 +105,13 @@ def extract_method(node)
           method_name
         end
 
+        # Only add the method to the chain if the method being
+        # called if it's a member of the time class.
+        def add_to_chain(node)
+          receiver, _method_name, *_args = *node
+          !(receiver.inspect.index 'Time').nil?
```

Comment:
Thanks for your feedback! I had a suspicion that this may not have been the best way to check it. Upon looking at more code, it looks like doing 
`receiver.source.include 'Time'` 
would be a better way to check it. I made this change in my most recent commit. What do you think?